lib/Digitick/Sepa/DomBuilder/BaseDomBuilder.php

```diff
@@ -185,7 +185,7 @@ public function getRemittenceElement($message)
      * @param string $creditorReference
      * @return \DOMElement
      */
-    public function getStructuredRemittanceElement($creditorReference)
+    public function getStructuredRemittanceElement($creditorReference, $creditorReferenceType = null)
```

Comment:
I think here, it would be good to pass the whole transactionInformation Instance instead of adding another parameter to the method